pygame_functions.py

```diff
@@ -6,6 +6,12 @@
 
 import pygame, sys, os
 
+from pathlib import Path
+DIR = Path(__file__).parent.absolute()
+DIR = f'{DIR}'.replace('\\','/')
+os.chdir(DIR)
+
+
```

Comment:
Changes file path to where the current program (i.e. pygame_functions) is. If it's in a different folder you can still edit DIR and change the directory. As a vscode user, this is necessary for me.